certified-connectors/JG Integrations/Readme.md

```diff
@@ -0,0 +1,17 @@
+# JG Integrations Flow Connector
+This connector was designed with our clients in mind however it provides a diverse range of tools that benefit all users.
+
```

Comment:
@jglassonnz You are missing a required section called "Publisher", Please make sure that you are following [our template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) for readme.md